Toolset/palettes/inspector/revinspector.livecodescript

```diff
@@ -99,7 +99,7 @@ on ideInspectObjects pObjects
    palette stack tTargetInspector
    unlock messages
    set the cSelectedObjects of stack tTargetInspector to tSelectedObjects
-   
+   set the cLocked of stack tTargetInspector to tSelectedObjects is not the selectedObjects
```

Comment:
This might have to be revIDESelectedObjects() instead of the selectedObjects - the latter is empty when there is just a focused stack, so this will cause stack inspectors to lock automatically won't it?